src/core/public/chrome/chrome_service.tsx

```diff
@@ -116,14 +122,25 @@ export class ChromeService {
     const helpSupportUrl$ = new BehaviorSubject<string>(KIBANA_ASK_ELASTIC_LINK);
     const isNavDrawerLocked$ = new BehaviorSubject(localStorage.getItem(IS_LOCKED_KEY) === 'true');
 
+    const formatKbnVersionHeader = () => {
+      if (!this.params.coreContext.env.packageInfo.version) return;
+      // question: What if the version is missing or corrupt?
+      const unformattedVersion = this.params.coreContext.env.packageInfo.version;
```

Comment:
I think we can safely guarantee that it's available. There are many Kibana features that would break if this isn't available (SO migrations, for one)